src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,64 @@
+package parser;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.testng.annotations.Test;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+
+public class JsonParserTest {
+
+    Cart artsiomCart = new Cart("artsiom-cart");
+    RealItem real = new RealItem();
+    Parser parser = new JsonParser();
```

Comment:
Couple of comments here:
1. Add access modifier. I recommend to avoid usage of default access modifier until you are sure that you need it :)
2. Let's have these 3 variables as class fields, but let's initialize them in method marked with @BeforeEach annotation

Please reach me if this comment is not clear

Reply:
it seems to be done